src/nameof.ts

```diff
@@ -32,7 +32,7 @@ export function nameof<T extends Object>(nameFunction: ((obj: T) => any) | { new
 
     // ES5 prop selector:
     // "function (x) { return x.prop; }"
-    if (fnStr.startsWith("function (")) {
+    if (fnStr.startsWith("function (") || fnStr.startsWith("function (")) {
```

Comment:
the added condition appears to duplicate the first. i think you meant to copy it and then remove the space between `function` and `(`?

Reply:
Actually, on second thought, these conditions can be reduced to one with something like `fnStr.trim().startsWith("function(")`.

Reply:
My bad - that would be `replace(/\s/g, "")` rather than `trim` (I believe - would have to double check that `\s` means "all whitespace", but I think it is).

Reply:
My bad. I made the change locally and forgot to save it. 

Performance wise I think `.replace()` is gonna be much more expensive. especially for a large function. 
```
(new RegExp(/function(\s)?\(/,"g")).test(fnStr) 
```

Isn't the above better?

Reply:
That does make sense. Good call!